src/shared/components/Footer/Footer.tsx

```diff
@@ -74,6 +74,7 @@ export const Footer: FunctionComponent<RouteComponentProps & UserProps> = ({
 						<a
 							href="https://meemoo.be/nl"
 							title={t('shared/components/footer/footer___meemoo')}
+							rel="noopener noreferrer"
```

Comment:
https://html.spec.whatwg.org/multipage/links.html#link-type-noreferrer

`noreferrer` heeft intern al de functionaliteit van `noopener` according to html specs, dus lijkt `noopener` mij redundant..